packages/craco/README.md

```diff
@@ -155,11 +244,7 @@ You can also change the location of the `craco.config.js` file by specifying the
 }
 ```
 
-## Configuration Overview
-
-When the property **mode** is available there are 2 possible values:
-- `extends`: the provided configuration will extends the CRA settings (**default mode**)
-- `file`: the CRA settings will be reseted and you will provide an official configuration file for the plugin ([postcss](https://github.com/michael-ciniawsky/postcss-load-config#postcssrc), [eslint](https://eslint.org/docs/user-guide/configuring#configuration-file-formats)) that will supersede any settings.
+## The CRACO Configuration File Template
```

Comment:
I understand why you might see it as a template. The original idea though is to showcase the available option not being a template.

This is a lazy alternative to write a more thorough documentation ;)

Reply:
Ah. Would it be helpful to put it in a different context than calling it a template?

Reply:
What do you have in mind?

Reply:
I changed the language to sample, and factored it into the beginner section. I describe it thus: 

> The [sample CRACO config file](https://github.com/banagale/craco/blob/172/note-about-devserver-config/packages/craco/(#sample-craco-configuration-file)) is meant to show possibilities for configuring your CRA-based project.